test/test-api-pass-error.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+// Testcase for passing an error object to the API call.
+
+if (process.argv[2] === 'child') {
+  const nodereport = require('../');
+  try {
+    throw new Error("Testing error handling");
+  } catch (err) {
+	  nodereport.triggerReport(err);
```

Comment:
@rnchamberlain what happened to https://github.com/nodejs/node-report/pull/46? It would prevent this.